.env.example

```diff
@@ -0,0 +1,38 @@
+# Default values for Azure OpenAI 
+AZURE_OPENAI_EMBEDDING_DEPLOYMENT_NAME=text-embedding-ada-002
+AZURE_OPENAI_EMBEDDING_MODEL_NAME=text-embedding-ada-002
+AZURE_OPENAI_PROMPT_DEPLOYMENT_NAME=gpt-35-turbo
+AZURE_OPENAI_PROMPT_MODEL_NAME=gpt-35-turbo
+AZURE_OPENAI_DEFAULT_ENDPOINT=https://griptape-cloud-nonprod-scus.openai.azure.com/
+
+# Get from Azure Portal https://azure.microsoft.com/en-us/get-started/azure-portal
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID=
+AZURE_CLIENT_SECRET=
+
+# Get key from https://platform.openai.com/account/api-keys
+OPENAI_API_KEY=<YOUR_API_KEY>
+
+# For cloud based testing: https://cloud.griptape.ai
+# For Skatepark testing: http://127.0.0.1:5000 
```

Comment:
Skatepark is actually configurable. Suggest you crib from example comment here: https://github.com/griptape-ai/managed-structure-template/blob/main/example-client/.env.client.example#L11